### PR TITLE
Feature/update sample env files

### PR DESCRIPTION
**Pull request by NatSquared**
*Description of changes:*
Update sample .env files and a few other development files to help get GDX devs set up on this branch

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -1,6 +1,20 @@
 # Change Log
+
 All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/).
 
+
+## v1.0.1 - 2023-10-18
+
+> **Feature**: Update sample .env files
```
Comment (jareth-whitney) [inline CHANGELOG.MD:8]: Do these updates have ticket numbers? I'm not sure if we're still doing this, but we used to add updates in this format:

* Added option to add a custom link for the Hub Update custom post type. ([DESENG-410](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-410))

File: met-api/sample.env [hunk 2]
```diff
@@ -1,52 +1,45 @@
...
+# EPIC integration configuration
+EPIC_URL=https://eagle-dev.apps.silver.devops.gov.bc.ca/api/commentperiod
+EPIC_JWT_OIDC_ISSUER=${KEYCLOAK_BASE_URL}/auth/realms/eao-epic
+EPIC_KC_CLIENT_ID=eagle-admin-console
+EPIC_MILESTONE=5cf00c03a266b7e1877504e9
+EPIC_KEYCLOAK_SERVICE_ACCOUNT_ID=
+EPIC_KEYCLOAK_SERVICE_ACCOUNT_SECRET=
+
+# Allowed CORS origins
+CORS_ORIGIN=http://localhost:3000,http://localhost:5000
```
Comment (jareth-whitney) [inline met-api/sample.env:45]: I love the organization here, thanks for fixing things up.

Are there any values in here that are sensitive and shouldn't be visible via GitHub?

File: met-web/sample.env [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+# The role needed to be considered an admin
+# TODO: Allocate a dedicated role for this on SSO
+REACT_APP_KEYCLOAK_ADMIN_ROLE=create_tenant
+
+# `met-api` endpoint
 REACT_APP_API_URL=http://localhost:5000/api
+
+# `analytics-api` endpoint
 REACT_APP_ANALYTICS_API_URL=http://localhost:5001/api
-REACT_APP_ENGAGEMENT_PROJECT_TYPES=Energy-Electricity,Energy - Petroleum & Natural Gas,Food Processing,Industrial,Mines,Other,Tourist Destination Resorts,Transportation,Waste Disposal,Water Management
```
Comment (jareth-whitney) [inline met-web/sample.env:8]: This data seemed out of place here. I assume it's been relocated to somewhere more appropriate?

**Review by jareth-whitney — APPROVED** [review]

File: met-web/sample.env (hunk 3 above)
Comment (NatSquared) [reply met-web/sample.env:8]: Currently the same data is defined as the "default" [elsewhere in the repository](https://github.com/bcgov/met-public/blob/274051ecf8f024ed47f8c45b5aca70182278d291/met-web/src/config.ts#L68). I think that should ALSO be moved somewhere else, but that's out of scope for this ticket :)

File: met-api/sample.env (hunk 2 above)
Comment (NatSquared) [reply met-api/sample.env:45]: Mostly the S3 secrets, and the MET_ADMIN variables. Once we get the client id/secret from Pathfinder SSO, that will also be sensitive.
However, none of that information is present in the GH release; the relevant variables were replaced with [blank definitions](https://github.com/bcgov/met-public/blob/274051ecf8f024ed47f8c45b5aca70182278d291/met-api/sample.env#L29).

File: CHANGELOG.MD (hunk 1 above)
Comment (NatSquared) [reply CHANGELOG.MD]: You're right; I linked the relevant ticket in [61145ce](https://github.com/bcgov/met-public/pull/2320/commits/61145cefa7c6850b1e1336dc59f0324f096e4f7c). Thanks!

**Pull request merged**